From 6216d62892cd2192a1fa8e539f68e5a216405d94 Mon Sep 17 00:00:00 2001
From: Xu Mao <B34688@freescale.com>
Date: Fri, 27 Dec 2013 10:33:32 +0800
Subject: [PATCH 278/304] ENGR00133115 remove HAVE_FSL_IMX_CODEC from device/

because it can be initialized in codec_env.mk
located in fsl_imx_omx or fsl_vpu_omx

Signed-off-by: Xu Mao <B34688@freescale.com>
---
 imx5x/BoardConfigCommon.mk | 2 +-
 imx6/soc/imx6dq.mk         | 3 ++-
 imx6/soc/imx6sl.mk         | 2 +-
 3 files changed, 4 insertions(+), 3 deletions(-)

diff --git a/imx5x/BoardConfigCommon.mk b/imx5x/BoardConfigCommon.mk
index c1914e6..6b15bd2 100755
--- a/imx5x/BoardConfigCommon.mk
+++ b/imx5x/BoardConfigCommon.mk
@@ -40,9 +40,9 @@ TARGET_HAVE_IMX_GRALLOC := true
 TARGET_HAVE_IMX_HWCOMPOSER := true
 USE_ION_ALLOCATOR := true
 
-HAVE_FSL_IMX_CODEC := true
 -include external/fsl_vpu_omx/codec_env.mk
 -include external/fsl_imx_omx/codec_env.mk
+
 BUILD_WITHOUT_FSL_DIRECTRENDER := false
 BUILD_WITHOUT_FSL_XEC := true
 
diff --git a/imx6/soc/imx6dq.mk b/imx6/soc/imx6dq.mk
index 98d55a4..8059fc8 100644
--- a/imx6/soc/imx6dq.mk
+++ b/imx6/soc/imx6dq.mk
@@ -9,9 +9,10 @@ HAVE_FSL_IMX_GPU3D := true
 HAVE_FSL_IMX_IPU := true
 BOARD_KERNEL_BASE := 0x10800000
 TARGET_KERNEL_DEFCONF := imx6_android_defconfig
-HAVE_FSL_IMX_CODEC := true
+
 -include external/fsl_vpu_omx/codec_env.mk
 -include external/fsl_imx_omx/codec_env.mk
+
 TARGET_HAVE_IMX_GRALLOC := true
 TARGET_HAVE_IMX_HWCOMPOSER = true
 USE_OPENGL_RENDERER := true
diff --git a/imx6/soc/imx6sl.mk b/imx6/soc/imx6sl.mk
index b40184e..f21c433 100644
--- a/imx6/soc/imx6sl.mk
+++ b/imx6/soc/imx6sl.mk
@@ -9,7 +9,7 @@ HAVE_FSL_IMX_GPU3D := false
 HAVE_FSL_IMX_IPU := false
 BOARD_KERNEL_BASE := 0x80800000
 TARGET_KERNEL_DEFCONF := imx6s_android_defconfig
-HAVE_FSL_IMX_CODEC := false
+-include external/fsl_vpu_omx/codec_env.mk
 TARGET_HAVE_IMX_GRALLOC := true
 TARGET_HAVE_IMX_HWCOMPOSER = true
 USE_OPENGL_RENDERER := false
-- 
1.8.0

